src/renderer/components/update-button/update-button.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import styles from "./styles.module.scss";
+
+import React, { useState } from "react";
+import { Menu, MenuItem } from "../menu";
+import uniqueId from "lodash/uniqueId";
+import { noop } from "lodash";
+import { cssNames } from "../../utils";
+
+interface UpdateButtonProps {
+  warningLevel?: "light" | "medium" | "high";
+  update: () => void;
+}
+
+export function UpdateButton({ warningLevel, update }: UpdateButtonProps) {
+  const id = uniqueId("update_button_");
```

Comment:
I don't particularly like the fact that this is non-deterministic. Can we just have `id` passed in as a required prop?

Reply:
It is just used by the following `<Menu>` to find element which will trigger menu to open. So no need to make it a required prop, I want to make this component to be as easy as possible.

Reply:
Yeah but it makes snapshots harder because the ID's will change if we reorder the tests.

Reply:
Maybe this? Saves only initially rendered `props.id` if provided or generating random one:
Or if we need to support dynamic `props.id` for component, then go with a getter or computed.

```suggestion
  const id = this.props.id ?? uniqueId("update_button_");
```